cmd/serve/serve.go

```diff
@@ -102,6 +103,18 @@ var ServeCmd = &cobra.Command{
 				}
 				return int32(topK)
 			}
+			maxTokens := func() int {
```

Comment:
Shouldn't we have a dedicated function? This function could be useful in CLI mode if the user wants to override the configuration with environment variables.

Moreover, I find that this improves readability.

Reply:
@AlexsJones can you comment

Reply:
I am okay with it like this, if we create an issue to refactor this file later on.